themes/demuxed-2022/template.html

```diff
@@ -228,12 +228,16 @@
 </style>
 
 <media-controller
+  breakpoints="xs:360 sm:600 md:760 lg:960 xl:1100"
+  nodefaultstore="{{nodefaultstore}}"
```

Comment:
How does someone even use `nodefaultstore` with a theme? They don't have access to the media controller object, right?

Reply:
they could get access via JS, I copied this from the microvideo theme 
https://github.com/muxinc/media-chrome/pull/784/files#diff-04d049643002288cedcc03079b40a0542a1e3b56bbf0eb57b1507f81d09ab870R418

I'll remove it from all then for now.